benchmark/README.md

```diff
@@ -15,3 +37,26 @@ bash scripts/configure-environment.sh
 ```
 
 * Follow the guidelines until installation is complete. 
+
+## Configure the inputs
+
+* You can configure following options in `/benchmark/config.json`
+
+| Option | Description | Default Value |
+| --- | --- | --- |
+| types | List of databases to test | cassandra,mysql,elasticsearch|
```

Comment:
What are the acceptable list of databases?

Reply:
currently Cassandra, MySQL, Elasticsearch are supported. added a note.